pkg/operator/bootstrap.go

```diff
@@ -61,6 +66,28 @@ func (c *Controller) Bootstrap() error {
 		mgmtState = operatorapi.Removed
 	}
 
+	infra, err := util.GetInfrastructure(c.listers)
+	if err != nil {
+		return err
+	}
+
+	rolloutStrategy := appsapi.RollingUpdateDeploymentStrategyType
+
+	// If Swift service is not available for OpenStack, we have to start using
+	// Cinder with RWO PVC backend. It means that we need to create an RWO claim
+	// and set the rollout strategy to Recreate.
+	switch infra.Status.PlatformStatus.Type {
+	case configapiv1.OpenStackPlatformType:
+		if swift.IsSwiftEnabled(c.listers) {
+			err = c.createPVC(corev1.ReadWriteOnce)
```

Comment:
Will this function fail if the PVC already exist? In some cases it might be pre-created (a rare scenario - the operator fails to create the custom resource and it runs Bootstrap the second time).

Reply:
yep, I updated to code to handle this situation properly